.pre-commit-config.yaml

```diff
@@ -20,8 +20,9 @@ repos:
     -   id: lint-frontend
         name: Linting of JS, Vue, SCSS and CSS files
         description: This hook handles all frontend linting for Kolibri
-        entry: yarn run lint-frontend
+        entry: yarn run lint-frontend:format
         language: system
+        files: \.(js|vue|scss|css)$
```

Comment:
I thought that by not setting `files` and having `pass_filesnames: false`, that the hook would just run `entry` once like we normally do when we use the npm script.

Does this change end up running `entry` for every file?

Reply:
Possibly, in which case I should activate `pass_filenames`.

Noting that I did this because after merging the earlier PR, I noticed that my pre-commit step was taking a very long time, even with Python files which shouldn't be touched by this hook at all.

Will investigate further here.

Reply:
This is the config we _don't_ want. I ran the hook with `verbose : true` to see what was happening

```
    hooks:
    -   id: lint-frontend
        name: Linting of JS, Vue, SCSS and CSS files
        description: This hook handles all frontend linting for Kolibri
        entry: yarn run lint-frontend:format
        language: system
        files: \.(js|vue|scss|css)$
        pass_filenames: true <- will call a `yarn run lint-frontend --write <expanded regex> for each file passing regex
        verbose: true
```

But I don't think adding the `files` field changes anything as long as `pass_filenames: false`. It just calls `lint-frontend:format` once, regardless of the `files` regex

Reply:
```suggestion
```